src/borg/remote.py

```diff
@@ -345,8 +345,11 @@ def open(self, path, create=False, lock_wait=None, lock=True, exclusive=None, ap
                     break
             else:
                 raise PathNotAllowed(path)
+        # borg init on borg serve --append-only does not create an append only repo,
+        # while borg init --append-only does, regardless of the --append-only flag for serve.
```

Comment:
maybe add some double quotes around the borg commands to make sentence easier to parse?